frontend/src/metabase/querying/components/DatePicker/DateOperatorPicker/DateOperatorPicker.tsx

```diff
@@ -1,14 +1,13 @@
 import { useMemo } from "react";
 
-import { Group, Stack } from "metabase/ui";
+import { Stack, Select } from "metabase/ui";
```

Comment:
The [padding around the pills](https://github.com/metabase/metabase/assets/6830683/b3d74681-71a3-440d-b6ec-3e2ad34f6dda) looks a bit out of place, IMO it [looks better without it](https://github.com/metabase/metabase/assets/6830683/42139ad2-63c5-4a4d-a71c-6dfe63a6cbf5).

Reply:
Spot on!
Thank you <3

Reply:
Agreed! Thanks!

Reply:
This was just another consequence of the different implementations between `RelativeDatePicker` vs `SimpleRelativeDatePicker`.

The former lets its children deal with the padding, while the latter has a padding on the parent/container.

Reply:
Fixed in https://github.com/metabase/metabase/pull/44554/commits/1d4876bb2261d34922c33206619653c454ad5cc3

![image](https://github.com/metabase/metabase/assets/31325167/7b005a96-b2ed-4604-b460-1a80efc8383f)

![image](https://github.com/metabase/metabase/assets/31325167/a1071d2b-ca82-421c-86f7-eaa2faac9f4b)